tailwind.config.js

```diff
@@ -36,6 +36,15 @@ module.exports = {
     },
     colors,
     zIndex,
+    lineHeight: {
+      ...lineHeight,
```

Comment:
question(blocking): could we use [`extend: { }`](https://tailwindcss.com/docs/theme#extending-the-default-theme) instead of importing all the default options?

Reply:
I tried it with extend but it seems like we can't access these entries via apply in our main.scss